Sources/UAPusher/Builder/Audience.swift

```diff
@@ -0,0 +1,63 @@
+import JSON
+
+/// Audience segment of an Urban Airship push notification payload
+/// See: https://docs.urbanairship.com/api/ua/#audience-selectors
+public struct Audience: Segment {
+    
+    // MARK: Class fields
+    
+    public let key: String = "audience"
+    public var payload: JSON
```

Comment:
Should we declare this as a constant? I think we should discourage changing this value after being initialized since it probably just adds confusion.

Reply:
good point